src/Profile/ProfileKeys.cpp

```diff
@@ -97,8 +97,8 @@ const char LanguageFile[] = "LanguageFile"; // pL
 const char InputFile[] = "InputFile"; // pL
 const char PilotName[] = "PilotName";
 const char WeGlideEnabled[] = "WeGlideEnabled";
-const char WeGlidePilotID[] = "WeGlidePilotID";
-const char WeGlidePilotBirthDate[] = "WeGlidePilotBirthDate";
+const char WeGlideUserID[] = "WeGlideUserID";
+const char WeGlideUserBirthDate[] = "WeGlideUserBirthDate";
```

Comment:
Are you aware that such changes break existing configurations?

Reply:
Yes, I am. No one uses this code (and setting) outside of some developers, but since that's the nomenclature of objects on WeGlide, it's definitely a better name. It is better to straighten it out at the beginning than to work with different identifiers later!

Reply:
It's released in XCSoar 7.23, so people will start using it now.
If you break existing configurations, no matter if you think that's okay, that should at least be documented in the commit message. Not doing so is hiding it from reviewers. Don't hide stuff from me.

Reply:
I hoped we could get that done before release 7.23 - but I can see that that was a bit 'sporty' on my part ;-), because it was definitely in the starting blocks.

Reply:
By the way, I at least hinted at that in the commit message, but probably not clear enough that it directly affects the values ​​already set.
Should that also be mentioned in the NEWS.txt - or is it enough in the commit message?
A description in the manual is also still pending, but I didn't want to tackle that until everything was completely finished! At the moment there is still no dialog window before the upload about the confirmation and possible change of the set values ​​(user, aircraft, filename...). I've come a long way there, too, but I'll have to check it out more carefully before I submit it.

Reply:
Since version 7.23 has been released for some time - and I currently see no further progress on the way to a fast 7.24 - you are doing a lot to turn the OpenVario upside down! - I have decided to to leave the identifiers in the settings as they are already published.
In the end it's best that way: The WeGlide objects are called 'User', but they are used for 'Pilot' and 'Copilot' (and others)